baseband/vlbi_base/base.py

```diff
@@ -536,8 +536,8 @@ def write(self, data, invalid_data=False):
             Piece of data to be written, with sample dimensions as given by
             ``sample_shape``. This should be properly scaled to make best use
             of the dynamic range delivered by the encoding.
-        invalid_data : bool, optional
-            Whether the current data is invalid.  Default: `False`.
+        valid : bool, optional
+            Whether the current data is valid.  Default: `True`.
```

Comment:
Darn, mistake from me (probably): data is plural, so this should be "are valid".